doc/sphinx/users-guide/vcl-built-in-subs.rst

```diff
@@ -417,6 +422,11 @@ The `vcl_backend_response` subroutine may terminate with calling
     lookups hitting this object will be turned into passed transactions,
     as if ``vcl_recv`` had returned ``pass``.
 
+  ``error(status code, reason)``
+    Transition to :ref:`vcl_backend_error` with ``resp.status`` and
+    ``resp.reason`` being preset to the arguments of ``error()`` if
```

Comment:
We are dealing with `beresp.reason` in this context.

Reply:
Done